Source/NSStoryboard.m

```diff
@@ -0,0 +1,975 @@
+                                       stringValue: kind];
+                  
+                  [sbproxy addAttribute: pselector];
+                  [sbproxy addAttribute: ptarget];
+                  [sbproxy addAttribute: pident];
+                  [sbproxy addAttribute: psegueIdent];
+                  [sbproxy addAttribute: psender];
+                  [sbproxy addAttribute: pkind];
+                  NSUInteger count = [[objects children] count];
+                  [objects insertChild: sbproxy
+                               atIndex: count - 1];
+                  
+                  // add action to parent ONLY if it is NOT a controller..
+                  if (![segue_parent_parent_name isEqualToString: @"windowController"] &&
+                      ![segue_parent_parent_name isEqualToString: @"viewController"])
+                    {              
+                      // Create action...
+                      NSXMLElement *action = [NSXMLElement elementWithName: @"action"];
+                      NSXMLNode *selector
+                        = [NSXMLNode attributeWithName: @"selector"
+                                           stringValue: @"doAction:"];
+                      NSXMLNode *target
+                        = [NSXMLNode attributeWithName: @"target"
+                                           stringValue: pident_value];
+                      NSXMLNode *ident
+                        = [NSXMLNode attributeWithName: @"id"
+                                           stringValue: uid]; 
+                      [action addAttribute: selector];
+                      [action addAttribute: target];
+                      [action addAttribute: ident];
+                      [segue_parent addChild: action];
+                    }
+                  
+                  // Create the segue...
+                  NSStoryboardSegue *ss = [[NSStoryboardSegue alloc] initWithIdentifier: identifier
+                                                                                 source: src
+                                                                            destination: dst];
+                  [ss _setKind: kind];
+                  [ss _setRelationship: rel];
+                  
+                  // Add to maptable...
+                  [mapTable setObject: ss
+                               forKey: identifier];
+                  
+                } // only process segue objects...
+            } // iterate over objects in each set of connections
+        } // iterate over connection objs
+
+      // Add to cache...
+      [_documentsMap setObject: mapTable
+                        forKey: uuidString];
+      
+    }  // if connections > 0
+  return mapTable;
+}
+
+// Private instance methods...
+- (id) initWithName: (NSStoryboardName)name
+             bundle: (NSBundle *)bundle
+{
+  self = [super init];
+  if (self != nil)
+    {
+      NSString *path = [bundle pathForResource: name
```

Comment:
Should we use a local autorelease pool here?

Reply:
Not sure.

Reply:
At this point I don't think so.